styles/base.css

```diff
@@ -1,56 +1,47 @@
 @import url("./fonts.css");
 
 :root {
-    --purple: #663190;
     --green: #88B440;
-    --teal: #01B4CB;
-    --navy: #223061;
-    --brand-primary-black: #1C1C1C;
-    --black: #000000;
     --white: #FFFFFF;
+    --brand-primary-cyan: #51B1C7;
```

Comment:
`teal` is now called `cyan`, and the color changed from `#01B4CB` to `#51B1C7`

The color is called `cyan` on Figma, so I'm changing it to `cyan` here as well.